src/js/components/molecules/color/color.ts

```diff
@@ -24,13 +28,17 @@ export default class Color extends Vue implements ColorProps {
     get styleIcon() {
         const styles = {};
 
-        if (this.background === '') {
-            styles['color'] = 'black';
+        styles['color'] = ColorUtil.lightness(this.color) >= 0.5 ? '#000' : '#FFF';
 
-            return styles;
+        return styles;
+    }
+
+    get color() {
+        const hexColorNoPrefix = (/^([A-Fa-f0-9]{6})/);
```

Comment:
Kinda lame this needs to be verified in the frontend. Could the backend not check if the value has or needs a #? Like integration service?

Reply:
Had a discussion about it, my opinion was exactly the same, maybe you can discuss this with @evdtempel? I suggest to leave this hack in the view model for now.